### PR TITLE
Suppor for elastic v5.

### PR DESCRIPTION
**Pull request by mcollina**
cc @dadoonet @sitkb @megastef please review.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,12 +4,11 @@ jdk:
 - openjdk7
```
Comment (dadoonet) [inline .travis.yml]: I think you will need jdk8 for ES 5

Comment (mcollina) [inline .travis.yml]: How can i ran ElasticSearch v5 on travis? I can't get the right java version anywhere it seems.

Comment (mcollina) [inline .travis.yml]: I've made it, if anybody needs it is in https://github.com/travis-ci/travis-ci/issues/4178#issuecomment-119321586.

**Comment by mcollina** [comment]
> Not a docker guy here but you may want to know that elastic has now official docker images for elasticsearch.

I am using the official ones. These are just docker-compose things to bring Elasticsearch and Kibana up.

**Pull request merged**